handbook/ops/bizops/index.md

```diff
@@ -19,11 +19,26 @@ TODO
 
 This page describes Sourcegraph's analytics function, our data sources, and how to use our data tools.
 
-To reach us, mention @ericbm in the most relevant channel (i.e. #marketing for a marketing-related question).
+To reach us, mention @business-team in the most relevant channel (i.e. #marketing for a marketing-related question).
 
-## Submitting a data request
+## Active product analytics dashboards
 
-**Projects:** Proposals to capture and report on new data are in [GitHub issues in the Sourcegraph analytics repository](https://github.com/sourcegraph/analytics/issues). Provide the following data to help us understand and prioritize your request:
+Dashboards are structured by focus area or goal instead of team. As we have more and more projects being worked on and tracked concurrently, this makes it easier for the product, engineering and business teams to track, link to, and simplify the lifecycle of dashboards. For example, we don't need to think about when it's time to deprecate a look from the code intel dashboard. When the goal is removed, the dashboard is archived.
+
+WIP
```

Comment:
This seems like the wrong place for the list of all dashboards to live to me... I imagine you don't want to come here and update it every time one is added or archived, right? Is there a page on Looker that you can link to that shows all active dashboards instead?

Reply:
Honestly don't really know the answer here yet. I think having it here would lead to it being more up-to-date than buried in Looker. I'm going to leave it here for now and see how it goes